main.go

```diff
@@ -70,6 +70,18 @@ func main() {
 
 	// Add flags related to this operator
 	v := version.Get()
+	logger := zap.New(zap.UseFlagOptions(&opts))
```

Comment:
This has been moved here so that it logs the versions even when a Kubernetes isn't available.